plugins/drng/plugin.go

```diff
@@ -0,0 +1,87 @@
+package drng
+
+import (
+	"encoding/hex"
+
+	"github.com/iotaledger/goshimmer/packages/binary/drng"
+	"github.com/iotaledger/goshimmer/packages/binary/drng/payload"
+	"github.com/iotaledger/goshimmer/packages/binary/drng/state"
+	cbPayload "github.com/iotaledger/goshimmer/packages/binary/drng/subtypes/collectiveBeacon/payload"
+	"github.com/iotaledger/goshimmer/packages/binary/messagelayer/message"
+	"github.com/iotaledger/goshimmer/packages/binary/messagelayer/tangle"
+	"github.com/iotaledger/goshimmer/plugins/config"
+	"github.com/iotaledger/goshimmer/plugins/messagelayer"
+	"github.com/iotaledger/hive.go/events"
+	"github.com/iotaledger/hive.go/logger"
+	"github.com/iotaledger/hive.go/marshalutil"
+	"github.com/iotaledger/hive.go/node"
+)
+
+const name = "DRNG" // name of the plugin
+
+var PLUGIN = node.NewPlugin(name, node.Enabled, configure, run)
+
+var (
+	Instance *drng.Instance
```

Comment:
I think we shouldn't expose instance of singletons like this from a plugin. I'd rather like we have a function ala `drngplugin.DRNG()` which returns the instance. Then there you can also setup with a `sync.Once` the DRNG for the first time.

This is what we do in HORNET: https://github.com/gohornet/hornet/blob/5fb58b12de6fb8349dc7d526d73656090ed77d4e/plugins/gossip/plugin.go#L48